src/decode-response/handle-rpc-error.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+var abi = require("augur-abi");
+var errors = require("../errors/codes");
+
+var handleRPCError = function (method, returns, response) {
+  var i, len, responseNumber;
+  if (response) {
```

Comment:
Same as above, recommend inverting the if into a guard clause:
```
if (!response) return response;
...
```
Reduces complexity and makes the code easier to read.  Same applies to all other places where an `if (...) return` could avoid nesting, I'll not comment on the rest though.